packages/gamut/src/FlexGrid/Col.tsx

```diff
@@ -54,19 +54,28 @@ function getClassNames(props: ColProps) {
 }
 
 export type ColSizing = {
+  /** Size of the column on screensizes greater than `xs` */
```

Comment:
Was this intentional?

```suggestion
  /** Size of the column on screen sizes greater than `xs` */
```